pkg/format/format.go

```diff
@@ -143,24 +165,31 @@ func findDrives() {
 		// this probably shouldn't be so hard
 		// but d.Mode()&os.ModeDevice == 0 doesn't work as expected
 		mode := d.Sys().(*syscall.Stat_t).Mode
+
 		if (mode & syscall.S_IFMT) != syscall.S_IFBLK {
 			continue
 		}
+
 		// ignore if it matches regexp
 		if ignoreExp.MatchString(d.Name()) {
 			continue
 		}
-		driveKeys = append(driveKeys, filepath.Join("/dev", d.Name()))
-	}
-	sort.Strings(driveKeys)
-	for _, d := range driveKeys {
-		drives[d] = true
+
+		driveKey := filepath.Join("/dev", d.Name())
+
+		// ignore if the deviceVar argument is defined and the driveKey doesn't match the deviceVar
+		if deviceVar != "" && driveKey != deviceVar {
+			continue
+		}
```

Comment:
if the `--device` argument exists (non-empty) and the `driveKey` does not match the specified device from `--device`, then skip this device; this causes us to follow the above pattern of ignoring devices, but as applied to the new `--device` argument in this case

Reply:
I assume that `--device` should be in the format `/dev/sda` then?